README.md

```diff
@@ -14,6 +14,7 @@ This is the temporary home for standards and guidance relating to software devel
   - [SQL Prompt](/guides/version_control_guidance.md)
   - [Style guide for standards](/guides/style_guide_for_standards.md)
   - [Version control guidance](/guides/version_control_guidance.md)
+  - [Docker guidance](/guides/docker_guidance.md)
```

Comment:
This should be in the list alphabetically

Reply:
Done